elasticsearch/run.sh

```diff
@@ -2,9 +2,22 @@
 
 set -euo pipefail
 
-secret_dir=/etc/elasticsearch/secret
-
 export KUBERNETES_AUTH_TRYKUBECONFIG="false"
+ES_REST_BASEURL=https://localhost:9200
+LOG_FILE=elasticsearch_connect_log.txt
+RETRY_COUNT=300		# how many times
+RETRY_INTERVAL=1	# how often (in sec)
+
+retry=$RETRY_COUNT
+max_time=$(( RETRY_COUNT * RETRY_INTERVAL ))	# should be integer
+timeouted=false
+
+mkdir -p /elasticsearch/$CLUSTER_NAME
+secret_dir=/etc/elasticsearch/secret
+[ -f $secret_dir/searchguard-node-key ] && ln -s $secret_dir/searchguard-node-key /elasticsearch/$CLUSTER_NAME/searchguard_node_key.key
```

Comment:
This is really for https://github.com/openshift/origin-aggregated-logging/pull/222